### PR TITLE
User see project where he is participant: Fix #28 #30

### LINKED ISSUE #28: As a user, I want to see all projects where I am a participant
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the UI-DSM
- [x] I have searched [open](https://github.com/RHEAGROUP/UI-DSM/issues) and [closed](https://github.com/RHEAGROUP/UI-DSM/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

When a user is connected, he should be able to see all projects where he is a participant
### System Configuration

- UI-DSM version:
  - [ ] UI-DSM:         
  - [ ] Other:              
- Environment (Operating system, version and so on):
- .NET Framework version:
- Additional information:

### PR DESCRIPTION
**Pull request by antoineatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/UI-DSM/pulls) open
- [x] I have verified that I am following the UI-DSM [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/UI-DSM/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description

Fix #28 #30 

- ProjectService retrieve projects where the user is involved
- ParticipantCreation component enables the administrator to create a new Participant into a Project

### CODE REVIEW

File: UI_DSM/UI_DSM.Client/Services/Administration/ParticipantService/IParticipantService.cs [hunk 1]
```diff
@@ -60,5 +60,13 @@ public interface IParticipantService
         /// <param name="participant">The <see cref="Participant" /> to delete</param>
         /// <returns>A <see cref="Task" /> with the <see cref="RequestResponseDto" /></returns>
         Task<RequestResponseDto> DeleteParticipant(Participant participant);
+
+        /// <summary>
+        ///     Gets a collection of <see cref="UserEntity" /> that can be used for the creation of <see cref="Participant" /> into a
+        ///     <see cref="Project" />
+        /// </summary>
+        /// <param name="projectId"></param>
+        /// <returns></returns>
```
Comment (lxatrhea) [inline UI_DSM/UI_DSM.Client/Services/Administration/ParticipantService/IParticipantService.cs:69]: Missing return value docs

File: UI_DSM/UI_DSM.Client/Services/Administration/ParticipantService/ParticipantService.cs [hunk 2]
```diff
@@ -132,5 +135,33 @@ public async Task<RequestResponseDto> DeleteParticipant(Participant participant)
                 throw new HttpRequestException(exception.Message);
             }
         }
+
+        /// <summary>
+        ///     Gets a collection of <see cref="UserEntity" /> that can be used for the creation of <see cref="Participant" /> into a
+        ///     <see cref="Project" />
+        /// </summary>
+        /// <param name="projectId"></param>
+        /// <returns></returns>
```
Comment (lxatrhea) [inline UI_DSM/UI_DSM.Client/Services/Administration/ParticipantService/ParticipantService.cs:144]: Missing return value docs

File: UI_DSM/UI_DSM.Client/ViewModels/Pages/Administration/ProjectPages/IProjectPageViewModel.cs [hunk 3]
```diff
@@ -27,6 +32,21 @@ public interface IProjectPageViewModel
         /// </summary>
         IProjectDetailsViewModel ProjectDetailsViewModel { get; }
 
+        /// <summary>
+        ///     Value indicating if the <see cref="DxPopup" /> for the creation of <see cref="Participant" /> is visible
+        /// </summary>
+        bool CreationPopupVisible { get; set; }
```
Comment (lxatrhea) [inline UI_DSM/UI_DSM.Client/ViewModels/Pages/Administration/ProjectPages/IProjectPageViewModel.cs:38]: A little bit religious, but a ViewModel should not "know" anything about its View. A state that describes that a Creation Popup is visible should be implemented from the ViewModel's point of view. So maybe the name should be like "LockedForEdit", or something like that. I don't know what it exactly does.

**Review by lxatrhea — APPROVED** [review]
Some docs and an advice.

**Review by omaratrhea — APPROVED** [review]

File: UI_DSM/UI_DSM.Client/ViewModels/Pages/Administration/ProjectPages/IProjectPageViewModel.cs (hunk 3 above)
Comment (antoineatrhea) [reply UI_DSM/UI_DSM.Client/ViewModels/Pages/Administration/ProjectPages/IProjectPageViewModel.cs]: @lxatrhea Are the changes ok for you?

**Review by lxatrhea — APPROVED** [review]

**Pull request merged**